grafana-plugin/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "description": "Grafana OnCall Plugin",
   "scripts": {
-    "lint": "eslint --cache --ext .js,.jsx,.ts,.tsx --max-warnings=0 ./src ./e2e-tests",
-    "lint:fix": "eslint --max-warnings=0 --fix --cache --ext .js,.jsx,.ts,.tsx ./src ./e2e-tests",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx --max-warnings=20 ./src ./e2e-tests",
+    "lint:fix": "eslint --max-warnings=20 --fix --ext .js,.jsx,.ts,.tsx ./src ./e2e-tests",
```

Comment:
There are some not obvious warnings that we will tackle separately. Until we're done with setting IRM up I think we should have some flexibility in eslint warnings to not block development bcs of that. Keep in mind those are just warnings, the eslint fails in case there is any error.

Reply:
agreed 👍